core/src/core/pmmr/readonly_pmmr.rs

```diff
@@ -28,7 +28,7 @@ where
 	B: Backend<T>,
 {
 	/// The last position in the PMMR
-	last_pos: u64,
+	size: u64,
```

Comment:
Like I said above, we now equate size with next available position.
We can't be troubled by implicit conversion since using a size as position requires calling one of the affected methods.